src/bundle/Resources/views/fieldtypes/edit/binary_base.html.twig

```diff
@@ -20,7 +22,10 @@
         {% set widget_wrapper_attr = widget_wrapper_attr|merge({'hidden': 'hidden'}) %}
     {% endif %}
 
-    {% set widget_container_block = block('binary_base_widget_container') %}
+    {% if not widget_container_block_name is defined %}
```

Comment:
Why not `{% if widget_container_block_name is not defined %}`? Looks better and reads better.